Added complex(a, b) signature

DionTimmermann opened the pull request:
This commit fixes issue #71.

This adds support for the following operations:
```MATLAB
complex(LinProp(1), 2)
complex(1, LinProp(2))
complex(LinProp([1 2 3]), 2)
complex(LinProp([1 2]), [3;4])
```
If `a` and `b` have incompatibel sizes, e.g. `complex(LinProp([1 2 3]), [1 2])`, a suitable error is thrown.

wollmich commented:
It would be nice if you add some tests in https://github.com/DionTimmermann/metas-unclib-matlab-wrapper-tests.

wollmich commented:
The error messages using MATLAB R2018a are different for `LinProp` and `double`:

```
>> complex(LinProp([1 2 3]), [1 2])
Arrays have incompatible sizes for this operation.
 
>> complex([1 2 3], [1 2])
Error using complex
Input arrays must have the same size.
```

And MATLAB R2018a does not allow the following operation for `double`;

```
>> complex([1 2], [3;4])
Error using complex
Input arrays must have the same size.
```

DionTimmermann commented:
> It would be nice if you add some tests in https://github.com/DionTimmermann/metas-unclib-matlab-wrapper-tests.

I will try to do so next week.

DionTimmermann commented:
I have added tests for `complex()` and `isreal()` to https://github.com/DionTimmermann/metas-unclib-matlab-wrapper-tests/commit/025e263c8b4a6ada16404e31285611c2ae552491. 

I have also updated the `complex()` method to fail if two arrays of unequal size are passed. (That case also produces an error in MATLAB 2021a - I had simply overlooked that case.)

For me, all tests in `test_complex.m` now pass, both in MATLAB 2018a and 2021a.
